lib/tarfiles.py

```diff
@@ -92,7 +93,7 @@ def do_tarballs(args: Dict[str, str]) -> None:
     """
     # pylint: disable=too-many-nested-blocks,too-many-branches
     res = []
-    clean_up = []
+    clean_up: List[Any] = []
```

Comment:
We know `clean_up` is a `List[str]` right, since `tar_up()` returns `str`?

Reply:
Yeah - it looks like mypy has a few open bugs where it confuses `bytes` and `strings` in certain cases.  See 
https://github.com/python/mypy/issues/12961
and 
https://github.com/python/mypy/issues/12666

This seems to be related, as the output of pre-commit if I make this and the below change is that I'm trying to use the product of an iteration (`str`) as a `byte` object (Line 145, 149).  So that's why I've set the type of `clean_up` to be `List[Any]` and ignored line 149 (which produces an identical error to the second issue I've linked above).

Reply:
Never mind.  Fixed.  It was type inference.  We used `tf` as a `bytes` earlier, and then when reassigning, used it as a `str` in a loop later (145), and mypy wasn't having that.

Reference:  https://mypy.readthedocs.io/en/stable/type_inference_and_annotations.html

So I made both this change and the one suggested below in [255129e](https://github.com/marcmengel/jobsub_lite/pull/78/commits/255129ebada73a564b2cabbad9733d5ff46bdfc8).

Reply:
Good catch!